datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -294,24 +292,24 @@ def normalize(self, metric, prefix=None, fix_case=False):
         name = re.sub(br"_\.", b".", name)
 
         if prefix is not None:
-            return prefix + "." + name
+            return ensure_bytes(prefix) + b"." + name
```

Comment:
Why isn't `prefix` getting the same treatment as the `metric` ?

Reply:
It wasn't before too. I'm just making sure everything is bytes here.

Reply:
it is a bit strange that we have inconsistent string handling in this function